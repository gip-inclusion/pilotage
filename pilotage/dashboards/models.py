from django.core.validators import MaxLengthValidator
from django.db import models


class Category(models.Model):
    class Meta:
        verbose_name = "catégorie"
        verbose_name_plural = "catégories"

    title = models.fields.CharField(verbose_name="titre", validators=[MaxLengthValidator(100)])
    display_order = models.fields.PositiveSmallIntegerField(verbose_name="ordre d'affichage", blank=True, default=0)

    def __str__(self):
        return f"{self.title}"


class Dashboard(models.Model):
    class Meta:
        verbose_name = "tableau de bord"
        verbose_name_plural = "tableaux de bord"

    title = models.fields.CharField(verbose_name="titre", validators=[MaxLengthValidator(150)])
    baseline = models.fields.CharField(validators=[MaxLengthValidator(250)])
    slug = models.fields.SlugField(unique=True)
    category = models.ForeignKey(
        Category,
        on_delete=models.DO_NOTHING,
        verbose_name="catégorie",
    )
    metabase_db_id = models.fields.IntegerField(verbose_name="metabase ID")
    embed_url = models.fields.URLField(
        verbose_name="embed URL",
        help_text="Si vous souhaitez ajouter une url d'iframe à la place d'un 'Metabase ID', "
        "vous devez de mettre 0 dans le 'Metabase ID' et ajouter une url ci-dessus",
        null=True,
        blank=True,
    )
    description = models.fields.TextField(null=True, blank=True)
    tally_popup_id = models.fields.CharField(
        verbose_name="tally popup ID", null=True, blank=True, validators=[MaxLengthValidator(10)]
    )
    tally_embed_id = models.fields.CharField(
        verbose_name="tally embed ID", null=True, blank=True, validators=[MaxLengthValidator(10)]
    )
    active = models.fields.BooleanField(verbose_name="actif", default=True)
    new = models.fields.BooleanField(verbose_name="nouveau", default=False)

    def __str__(self):
        return f"{self.title}"
