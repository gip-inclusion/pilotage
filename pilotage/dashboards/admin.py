from django.contrib import admin

from dashboards.models import Dashboard


class DashboardAdmin(admin.ModelAdmin):
    list_display = ('title', 'metabase_db_id')

admin.site.register(Dashboard, DashboardAdmin)
