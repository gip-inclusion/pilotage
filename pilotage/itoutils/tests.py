import re
from collections import defaultdict
from itertools import chain

from bs4 import BeautifulSoup


def remove_static_hash(content):
    return re.sub(r"\.[\da-f]{12}\.(svg|png|jpg)\b", r".\1", content)


def parse_response_to_soup(response, selector=None, no_html_body=False, replace_in_attr=None):
    soup = BeautifulSoup(response.content, "html5lib", from_encoding=response.charset or "utf-8")
    if no_html_body:
        # If the provided HTML does not contain <html><body> tags
        # html5lib will always add them around the response:
        # ignore them
        soup = soup.body
    if selector is not None:
        [soup] = soup.select(selector)
    title = soup.title
    if title:
        title.string = re.sub(r"\s+", " ", title.string)
    for csrf_token_input in soup.find_all("input", attrs={"name": "csrfmiddlewaretoken"}):
        csrf_token_input["value"] = "NORMALIZED_CSRF_TOKEN"
    if "nonce" in soup.attrs:
        soup["nonce"] = "NORMALIZED_CSP_NONCE"
    for csp_nonce_script in soup.find_all("script", {"nonce": True}):
        csp_nonce_script["nonce"] = "NORMALIZED_CSP_NONCE"
    for img in chain(
        soup.find_all("img", attrs={"src": True}), soup.find_all("input", attrs={"type": "image", "src": True})
    ):
        img["src"] = remove_static_hash(img["src"])
    for img in soup.find_all("source", attrs={"srcset": True}):
        img["srcset"] = remove_static_hash(img["srcset"])
    if replace_in_attr:
        replace_in_attr = list(replace_in_attr)
        # Get the list of the attrs (deduplicated) we should search for replacement
        attr_replacements = defaultdict(list)
        for attr, from_str, to_str in replace_in_attr:
            attr_replacements[attr].append((from_str, to_str))
        for attr, replacements in attr_replacements.items():
            nodes = (
                # Search and replace in descendant nodes
                *soup.find_all(attrs={attr: True}),
                # Also replace attributes in the top node
                *([soup] if attr in soup.attrs else []),
            )
            for node in nodes:
                for from_str, to_str in replacements:
                    node.attrs.update({attr: node.attrs[attr].replace(from_str, to_str)})
    return soup
