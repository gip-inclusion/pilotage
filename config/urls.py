from django.conf import settings
from django.contrib import admin
from django.urls import include, path
from pilotage.pilotage import urls as pilotage_urls
from pilotage.dashboards import urls as dashboards_urls

urlpatterns = [
    path(settings.PATH_TO_ADMIN, admin.site.urls),
    path("", include(pilotage_urls)),
    path("", include(dashboards_urls)),
]
