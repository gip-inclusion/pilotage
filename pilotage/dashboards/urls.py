from django.contrib import admin
from django.urls import path
from dashboards import views

app_name = "dashboards"
urlpatterns = [
    path(
        "tableaux-de-bord/",
        views.tableaux_de_bord_publics,
        name="tableaux_de_bord_publics",
    ),
    path(
        "tableaux-de-bord/<slug:slug>/",
        views.tableau_de_bord_public,
        name="tableau_de_bord_public",
    ),
]
