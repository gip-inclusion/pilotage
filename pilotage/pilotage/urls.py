"""
URL configuration for pilotage project.

The `urlpatterns` list routes URLs to views. For more information please see:
    https://docs.djangoproject.com/en/4.2/topics/http/urls/
Examples:
Function views
    1. Add an import:  from my_app import views
    2. Add a URL to urlpatterns:  path('', views.home, name='home')
Class-based views
    1. Add an import:  from other_app.views import Home
    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
Including another URLconf
    1. Import the include() function: from django.urls import include, path
    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
"""
from django.contrib import admin
from django.urls import include, path
from . import views

app_name = 'pilotage'
urlpatterns = [
    path('admin/', admin.site.urls),
    path('', views.home, name="home"),
	path('dashboards/', include('dashboards.urls')),
    path('stats/', views.stats, name="statistiques"),
    path('accessibilite/', views.accessibilite, name="accessibilite"),
    path('mentions-legales/', views.mentions_legales, name="mentions_legales"),
    path('politique-de-confidentialite/', views.politique_de_confidentialite, name="politique_de_confidentialite"),
    path('inscription-lettre-information/', views.inscription_lettre_information, name="inscription_lettre_information"),
    path('tableaux-de-bord-prives/', views.tableaux_de_bord_prives, name="tableaux_de_bord_prives"),
]
