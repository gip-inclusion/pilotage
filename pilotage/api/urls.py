from django.urls import path

from pilotage.api import views


app_name = "api"
urlpatterns = [
    path("dataset/<slug:name>", views.dataset, name="dataset"),
]
