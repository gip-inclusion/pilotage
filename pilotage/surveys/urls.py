from django.urls import path

from pilotage.surveys import views


app_name = "surveys"
urlpatterns = [
    path("<slug:survey_name>/", views.start, name="start"),
    path("<slug:survey_name>/<uuid:answer_uid>/<str:step>", views.tunnel, name="tunnel"),
]
