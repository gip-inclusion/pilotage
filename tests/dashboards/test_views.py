from django.core import management
from django.urls import reverse
from freezegun import freeze_time

from pilotage.itoutils.tests import parse_response_to_soup
from tests.dashboards.factories import DashboardFactory


def test_tableaux_de_bord_publics_without_dashboards(snapshot, client):
    assert (
        str(parse_response_to_soup(client.get(reverse("dashboards:tableaux_de_bord_publics")), selector="#main"))
        == snapshot()
    )


def test_tableaux_de_bord_publics_with_fixture_dashboards(snapshot, client):
    management.call_command(
        "loaddata",
        "pilotage/fixtures/dashboard.json",
    )
    assert (
        str(parse_response_to_soup(client.get(reverse("dashboards:tableaux_de_bord_publics")), selector="#main"))
        == snapshot()
    )


@freeze_time("2025-03-06")
def test_tableau_de_bord_public(snapshot, client):
    dashboard = DashboardFactory(
        for_snapshot=True,
    )

    assert (
        str(
            parse_response_to_soup(
                client.get(reverse("dashboards:tableau_de_bord_public", kwargs={"slug": dashboard.slug})),
                selector="#main",
            )
        )
        == snapshot()
    )
