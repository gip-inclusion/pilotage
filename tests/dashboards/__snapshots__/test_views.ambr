# serializer version: 1
# name: test_tableau_de_bord_public
  '''
  <main class="s-main" id="main" role="main">
              
                  
      <section>
          <div class="container">
              <div class="row">
                  <div class="col-12">
                      <nav aria-label="breadcrumb" class="c-breadcrumb">
                          <ol class="breadcrumb">
                              <li class="breadcrumb-item">
                                  <a href="/">Accueil</a>
                              </li>
                              <li class="breadcrumb-item">
                                  <a href="/tableaux-de-bord/">Statistiques publiques</a>
                              </li>
                              <li aria-current="page" class="breadcrumb-item active">super duper title</li>
                          </ol>
                      </nav>
                  </div>
              </div>
          </div>
      </section>
  
      <section class="s-title-01">
          <div class="s-title-01__container container">
              <div class="s-title-01__row row">
                  <div class="s-title-01__col col-12 col-lg-12">
                      <h1>
                          super duper title
                          
                      </h1>
                      <div><p>super duper description</p></div>
                      <!-- Quick win crassou pour permettre de modifier l'encart du TB 408 -->
                      
                  </div>
              </div>
          </div>
      </section>
  
      
          <section class="s-embed m-0">
              <div class="s-embed__container container">
                  <div class="s-embed__row row">
                      <div class="col-12">
                          <iframe allowtransparency="" id="iframeMetabase" src="https://metabase.fake/embed/dashboard/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyZXNvdXJjZSI6eyJkYXNoYm9hcmQiOjB9LCJwYXJhbXMiOnt9LCJleHAiOjE3NDEyMTk4MDB9.eog3uHiTf9_syfFz2kUub0FUfLapUrCF5VhUGLOmdTg#bordered=true&amp;titled=false" title="super duper baseline"></iframe>
                      </div>
                  </div>
              </div>
          </section>
      
  
      
          <section class="s-embed s-embed--tally">
              <div class="s-embed__container container">
                  <div class="s-embed__row row">
                      <div class="s-embed__col col-12">
                          <iframe data-tally-src="//tally.so/embed/5up3rdup3r?alignLeft=1&amp;hideTitle=1&amp;transparentBackground=1&amp;dynamicHeight=1" height="189" loading="lazy" title="Aidez-nous à améliorer votre outil !" width="100%"></iframe>
                      </div>
                  </div>
              </div>
          </section>
      
  
              
          </main>
  '''
# ---
# name: test_tableaux_de_bord_publics_with_fixture_dashboards
  '''
  <main class="s-main" id="main" role="main">
              
                  
      <section>
          <div class="container">
              <div class="row">
                  <div class="col-12">
                      <nav aria-label="breadcrumb" class="c-breadcrumb">
                          <ol class="breadcrumb">
                              <li class="breadcrumb-item">
                                  <a href="/">Accueil</a>
                              </li>
                              <li aria-current="page" class="breadcrumb-item active">Statistiques publiques</li>
                          </ol>
                      </nav>
                  </div>
              </div>
          </div>
      </section>
  
      <section class="s-title-01">
          <div class="s-title-01__container container">
              <div class="s-title-01__row row">
                  <div class="s-title-01__col col-12 col-xl-11">
                      <h1>Statistiques publiques</h1>
                      <p>Le pilotage de l'inclusion vise à résoudre les problématiques métier des professionnels sur le terrain et de leurs partenaires par la donnée. Cette approche se matérialise principalement à travers des tableaux de bord comprenant des données actualisées en temps réel. Ces outils, qu'ils soient stratégiques, opérationnels ou de suivi, s'adressent à tous les acteurs de l'inclusion. Les informations peuvent être affinées pour un territoire spécifique, tel qu'un bassin d'emploi, un type d'organisation comme une entreprise d'insertion ou un conseil départemental, et sur une période donnée. Grâce à ces connaissances, les utilisateurs peuvent ainsi suivre leurs progrès, se comparer, améliorer leurs échanges avec les financeurs et les partenaires, et enrichir leurs plans d'action et feuilles de route.</p>
                      <p class="h4">
                          Plaquette présentant l’ensemble des tableaux de bord développés par le Pilotage de l’inclusion
                          <a aria-label="Plaquette présentant l’ensemble des tableaux de bord" class="has-external-link text-nowrap" href="https://www.canva.com/design/DAF_5XyJqRk/fV5Rpv649EU5P0GJCo0VJQ/view?utm_content=DAF_5XyJqRk&amp;utm_campaign=designshare&amp;utm_medium=link&amp;utm_source=editor" target="_blank">disponible ici</a>.
                      </p>
                  </div>
              </div>
              <div class="s-title-01__row row">
                  <div class="s-title-01__col col-12">
                      <div class="alert alert-info mt-5" role="status">
                          <div class="row">
                              <div class="col-auto pe-0">
                                  <i aria-hidden="true" class="ri-information-line ri-xl text-info"></i>
                              </div>
                              <div class="col">
                                  <p class="mb-2">
                                      <strong>Venez découvrir le nouveau tableau de bord “Suivi des bénéficiaires, taux d’encadrement et présence en emploi” lors du webinaire du 2 juillet 2025 !</strong>
                                  </p>
                                  <p class="mb-0">Ce nouveau tableau de bord permet d’obtenir des statistiques sur les profils des personnes en parcours en SIAE, sur les taux d’encadrement et sur les taux de présence en emploi. Il sera présenté lors d’un webinaire de 45 minutes, le 2 juillet 2025 à 14h30.</p>
                              </div>
                              <div class="col-12 col-md-auto mt-3 mt-md-0 d-flex align-items-center justify-content-center">
                                  <a class="btn btn-sm btn-primary" href="https://bfv4l.r.bh.d.sendibt3.com/mk/cl/f/sh/7nVU1aA6Qs4xyddr7jLEI5zJTDDBH5a/RhtHBPIyqIDu" target="_blank">Inscrivez-vous !</a>
                              </div>
                          </div>
                      </div>
                  </div>
              </div>
          </div>
      </section>
  
      
      
          <section class="s-tableaux">
              <div class="s-tableaux__container container">
                  <div class="s-tableaux__row row mb-3">
                      <div class="col-12">
                          <h2 class="text-secondary">L'offre d'insertion par l'activité économique (IAE)</h2>
                      </div>
                  </div>
                  <div class="s-tableaux__row row">
                      <div class="s-tableaux__col s-tableaux__col--grid col-12">
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Métiers
                                          
                                      </h3>
                                      <p>Quels sont les métiers proposés et recherchés depuis fin 2019 ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/metiers/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Postes en tension
                                          
                                      </h3>
                                      <p>Quels sont les fiches de poste en tension ? Quelles situations dans les territoires ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/postes-en-tension/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Zoom sur les employeurs
                                          
                                      </h3>
                                      <p>Quels employeurs inclusifs dynamisent mon territoire et comment ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/zoom-employeurs/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                      </div>
                  </div>
              </div>
          </section>
      
          <section class="s-tableaux">
              <div class="s-tableaux__container container">
                  <div class="s-tableaux__row row mb-3">
                      <div class="col-12">
                          <h2 class="text-secondary">Les publics dans l'IAE</h2>
                      </div>
                  </div>
                  <div class="s-tableaux__row row">
                      <div class="s-tableaux__col s-tableaux__col--grid col-12">
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          La représentation des femmes dans les candidatures vers l'IAE
                                          
                                      </h3>
                                      <p>Quelle est la place des femmes dans l’IAE ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/femmes-iae/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                      </div>
                  </div>
              </div>
          </section>
      
          <section class="s-tableaux">
              <div class="s-tableaux__container container">
                  <div class="s-tableaux__row row mb-3">
                      <div class="col-12">
                          <h2 class="text-secondary">Les candidatures vers l'IAE</h2>
                      </div>
                  </div>
                  <div class="s-tableaux__row row">
                      <div class="s-tableaux__col s-tableaux__col--grid col-12">
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Bilan annuel des candidature en IAE
                                          <span class="badge badge-sm rounded-pill bg-important">Nouveau</span>
                                      </h3>
                                      <p>Quel bilan annuel faire sur l’offre en IAE, les candidats, les candidatures et les réalisations des prescripteurs ? Où en sont les régions et les départements ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/bilan-candidatures-iae/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Cartographies des orientations vers l'IAE
                                          
                                      </h3>
                                      <p>L'IAE dans les territoires, quelles données locales ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/cartographies-iae/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          État, suivi et résultat des candidatures orientées
                                          
                                      </h3>
                                      <p>Combien de candidatures orientées chaque mois ? Quels sont les résultats ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/etat-suivi-candidatures/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          L'auto-prescription
                                          
                                      </h3>
                                      <p>Comment les structures se saisissent de l’auto-prescription ? Quels critèrent remplissent les personnes recrutées en auto-prescription ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/auto-prescription/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Zoom sur les prescripteurs
                                          
                                      </h3>
                                      <p>Quels prescripteurs agissent sur mon territoire et comment ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/zoom-prescripteurs/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          L'accompagnement des prescripteurs habilités
                                          
                                      </h3>
                                      <p>Quel est le taux de transformation des prescripteurs habilités ? L’analyse porte sur les candidats pour lesquels les prescripteurs habilités ont réalisé le diagnostic d’éligibilité.</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/prescripteurs-habilites/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                      </div>
                  </div>
              </div>
          </section>
      
          <section class="s-tableaux">
              <div class="s-tableaux__container container">
                  <div class="s-tableaux__row row mb-3">
                      <div class="col-12">
                          <h2 class="text-secondary">Le pilotage du dispositif IAE</h2>
                      </div>
                  </div>
                  <div class="s-tableaux__row row">
                      <div class="s-tableaux__col s-tableaux__col--grid col-12">
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Conventionnements IAE
                                          
                                      </h3>
                                      <p>Combien d’ETP sont affectés aux structures de l’IAE et combien sont consommés ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/conventionnements-iae/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Analyses autour des conventionnements IAE
                                          
                                      </h3>
                                      <p>Quel part du budget de l’IAE bénéficie aux femmes ? Combien d’heures sont réalisées en moyenne dans les structures ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/analyses-conventionnements-iae/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Suivi du contrôle a posteriori
                                          
                                      </h3>
                                      <p>Quels sont les résultats des campagnes de contrôle a posteriori ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/suivi-controle-a-posteriori/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Demandes de prolongation
                                          
                                      </h3>
                                      <p>Combien de demandes de prolongation sont réalisées, et par qui ? Combien sont acceptées ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/suivi-demandes-prolongation/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Suivi des Pass IAE
                                          
                                      </h3>
                                      <p>Combien de Pass IAE arrivent à expiration, semaine après semaine ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/suivi-pass-iae/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                      </div>
                  </div>
              </div>
          </section>
      
          <section class="s-tableaux">
              <div class="s-tableaux__container container">
                  <div class="s-tableaux__row row mb-3">
                      <div class="col-12">
                          <h2 class="text-secondary">Les établissements et service d'accompagnement par le travail (ESAT)</h2>
                      </div>
                  </div>
                  <div class="s-tableaux__row row">
                      <div class="s-tableaux__col s-tableaux__col--grid col-12">
                          
                              <div class="d-flex flex-column justify-content-between position-relative">
                                  <div>
                                      <h3>
                                          Zoom sur les ESAT
                                          
                                      </h3>
                                      <p>Plan de transformation des ESAT : d’où partons-nous ?</p>
                                  </div>
                                  <p class="s-tableaux__col__cta">
                                      <a class="btn btn-outline-primary stretched-link" href="/tableaux-de-bord/zoom-esat/" rel="noopener">Voir le tableau de bord</a>
                                  </p>
                              </div>
                          
                      </div>
                  </div>
              </div>
          </section>
      
  
              
          </main>
  '''
# ---
# name: test_tableaux_de_bord_publics_without_dashboards
  '''
  <main class="s-main" id="main" role="main">
              
                  
      <section>
          <div class="container">
              <div class="row">
                  <div class="col-12">
                      <nav aria-label="breadcrumb" class="c-breadcrumb">
                          <ol class="breadcrumb">
                              <li class="breadcrumb-item">
                                  <a href="/">Accueil</a>
                              </li>
                              <li aria-current="page" class="breadcrumb-item active">Statistiques publiques</li>
                          </ol>
                      </nav>
                  </div>
              </div>
          </div>
      </section>
  
      <section class="s-title-01">
          <div class="s-title-01__container container">
              <div class="s-title-01__row row">
                  <div class="s-title-01__col col-12 col-xl-11">
                      <h1>Statistiques publiques</h1>
                      <p>Le pilotage de l'inclusion vise à résoudre les problématiques métier des professionnels sur le terrain et de leurs partenaires par la donnée. Cette approche se matérialise principalement à travers des tableaux de bord comprenant des données actualisées en temps réel. Ces outils, qu'ils soient stratégiques, opérationnels ou de suivi, s'adressent à tous les acteurs de l'inclusion. Les informations peuvent être affinées pour un territoire spécifique, tel qu'un bassin d'emploi, un type d'organisation comme une entreprise d'insertion ou un conseil départemental, et sur une période donnée. Grâce à ces connaissances, les utilisateurs peuvent ainsi suivre leurs progrès, se comparer, améliorer leurs échanges avec les financeurs et les partenaires, et enrichir leurs plans d'action et feuilles de route.</p>
                      <p class="h4">
                          Plaquette présentant l’ensemble des tableaux de bord développés par le Pilotage de l’inclusion
                          <a aria-label="Plaquette présentant l’ensemble des tableaux de bord" class="has-external-link text-nowrap" href="https://www.canva.com/design/DAF_5XyJqRk/fV5Rpv649EU5P0GJCo0VJQ/view?utm_content=DAF_5XyJqRk&amp;utm_campaign=designshare&amp;utm_medium=link&amp;utm_source=editor" target="_blank">disponible ici</a>.
                      </p>
                  </div>
              </div>
              <div class="s-title-01__row row">
                  <div class="s-title-01__col col-12">
                      <div class="alert alert-info mt-5" role="status">
                          <div class="row">
                              <div class="col-auto pe-0">
                                  <i aria-hidden="true" class="ri-information-line ri-xl text-info"></i>
                              </div>
                              <div class="col">
                                  <p class="mb-2">
                                      <strong>Venez découvrir le nouveau tableau de bord “Suivi des bénéficiaires, taux d’encadrement et présence en emploi” lors du webinaire du 2 juillet 2025 !</strong>
                                  </p>
                                  <p class="mb-0">Ce nouveau tableau de bord permet d’obtenir des statistiques sur les profils des personnes en parcours en SIAE, sur les taux d’encadrement et sur les taux de présence en emploi. Il sera présenté lors d’un webinaire de 45 minutes, le 2 juillet 2025 à 14h30.</p>
                              </div>
                              <div class="col-12 col-md-auto mt-3 mt-md-0 d-flex align-items-center justify-content-center">
                                  <a class="btn btn-sm btn-primary" href="https://bfv4l.r.bh.d.sendibt3.com/mk/cl/f/sh/7nVU1aA6Qs4xyddr7jLEI5zJTDDBH5a/RhtHBPIyqIDu" target="_blank">Inscrivez-vous !</a>
                              </div>
                          </div>
                      </div>
                  </div>
              </div>
          </div>
      </section>
  
      
      
  
              
          </main>
  '''
# ---
